src/test/librbd/mock/MockWatcher.h

```diff
@@ -0,0 +1,33 @@
+// -*- mode:C; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#ifndef CEPH_TEST_LIBRBD_MOCK_WATCHER_H
+#define CEPH_TEST_LIBRBD_MOCK_WATCHER_H
+
+#include "librbd/Watcher.h"
+#include "gmock/gmock.h"
+
+class Context;
+
+namespace librbd {
+
+struct MockWatcher : public Watcher {
```

Comment:
Thanks! Until now I didn't know why the journal TypeTraits was needed. Now I understand :-)